src/main/java/com/researchspace/dao/hibernate/RecordDaoHibernate.java

```diff
@@ -299,7 +299,7 @@ public List<RSpaceDocView> getRecordViewsById(Set<Long> dbids) {
   }
 
   @Override
-  public Set<BaseRecord> getViewableRecordsForUser(User user) {
+  public Set<BaseRecord> getViewableRecordsForUsers(List<User> users) {
```

Comment:
Could we use a `Set<User>` as an actual parameter instead? I guess that would save uys to re-map it at the end?

Reply:
Yes good shout will update this